integration/rpc.go

```diff
@@ -0,0 +1,43 @@
+package integration
+
+import (
+	"github.com/kaspanet/kaspad/util"
+	"github.com/kaspanet/kaspad/util/daghash"
+	"github.com/kaspanet/kaspad/wire"
+
+	rpcclient "github.com/kaspanet/kaspad/rpc/client"
+)
+
+type rpcClient struct {
+	*rpcclient.Client
+	onBlockAdded   func(*wire.BlockHeader)
```

Comment:
I would put here a channel instead, to save you from creating a channel later

Reply:
Not all tests are going to use that functionality, and if I create a channel all the time, I'll have to deal with bullshit such as making sure the channels are emptied, etc.